core/src/test/scala/com/pingcap/tispark/convert/ToDecimalSuite.scala

```diff
@@ -301,15 +309,14 @@ class ToDecimalSuite extends BaseDataSourceTest("test_data_type_convert_to_decim
         )
 
         val readA1: java.math.BigDecimal = java.math.BigDecimal.valueOf(11, 1)
-        val readA3: java.math.BigDecimal = java.math.BigDecimal.valueOf(1100, 3)
-        val readA5: java.math.BigDecimal = java.math.BigDecimal.valueOf(110000, 5)
-        val readB2: java.math.BigDecimal = java.math.BigDecimal.valueOf(-220, 2)
-        val readB4: java.math.BigDecimal = java.math.BigDecimal.valueOf(-22000, 4)
-        val readB6: java.math.BigDecimal = java.math.BigDecimal.valueOf(-2200000, 6)
+        val readA2: java.math.BigDecimal = java.math.BigDecimal.valueOf(-220, 2)
+        val readA3: java.math.BigDecimal = java.math.BigDecimal.valueOf(12, 3)
+        val readA4: java.math.BigDecimal = java.math.BigDecimal.valueOf(0, 4)
```

Comment:
is there any reason why A4 is (0, 4)?

Reply:
if you write `1.4e-25d` to column  `DECIMAL(38, 4)`, it will be `0.0000`

Reply:
while A5 is (0, 5)?